Included additional troubleshooting guidance (#7236)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/sit-get-started-exact-data-match-test.md

- If the SIT you selected for a primary element in the EDM type doesn't find a match in the item or finds fewer matches than you expected, check that it supports the separators and delimiters that are in the content. Be sure to include the ignored delimiters defined in your schema.

> - If the SIT you selected for a primary element in the EDM type does find matches in the item but those matches are not found when testing the EDM Sensitive information type itself, it may be that the sensitive information type you defined is starting or ending the match at the wrong point in the text. The two most common causes for this are:

Reviewer: ```suggestion
- If the SIT you selected for a primary element in the EDM type does find matches in the item but those matches are not found when testing the EDM sensitive information type itself, it may be that the sensitive information type you defined is starting or ending the match at the wrong point in the text. The two most common causes for this are:
```

>     1. Using a regular expression that starts or ends with a capturing whitespace delimiter, for example /s. If you do this, text matching your regular expression will include whitespace at the beginning and/or end of the captured string, and when that text is passed to EDM for matching, the hash of the text with the whitespace won't match the hash of the text entered in your table. To address this problem, you should use a word delimiter such as /b instead of /s, which will not include the whitespace as part of the captured string. 

Reviewer: ```suggestion
    1. Using a regular expression that starts or ends with a capturing whitespace delimiter, for example, /s. If you do this, text matching your regular expression will include whitespace at the beginning and/or end of the captured string, and when that text is passed to EDM for matching, the hash of the text with the whitespace won't match the hash of the text entered in your table. To address this problem, you should use a word delimiter such as /b instead of /s, which will not include the whitespace as part of the captured string. 
```

    2. Using a regular expression that matches a substring of the actual string you want to find. For example, a simple pattern for detecting email addresses might look like [a-zA-Z]{30}@[a-zA-Z]{20}.[a-zA-Z]{2,3}. Such a regular expression would correctly match an email address like john@contoso.com, but it would match an address with a multi-level domain name such as john@contoso.co.jp incorrectly since it would only pick the text until the .com domain and miss the country specific domain part, causing the capture of a string that doesn't correspond to what is in your sensitive information table. You can address this by using a regular expression that is designed to match the whole string you want to capture.

> - If the EDM SIT detects content when secondary evidence is not required but fails to detect content when secondary evidence requirements are included, or detects fewer matches than expected, make sure values for secondary evidence you are trying to detect are composed of a single "token" (a word or string that does not contain spaces in it). For example, first or last names can be single or multi-token strings since each can be composed of multiple words. When matching additional evidence EDM will, by default, match individual tokens in the text, so multi-token strings will not be detected. To address this, you can define a sensitive information type that finds matches to such multi-token strings (e.g. using a regular expression such as \b[A-Z][a-z]{1,25}([ -][A-Z][a-z]{1,25}){0,4}\b which will match any sequence of one0 to five consecutive words that start with an uppercase character), and use this sensitive information type as the "Classification" element for the additional evidence conditions in your EDM sensitive info type XML using the same syntax used for the primary element. See https://docs.microsoft.com/en-us/microsoft-365/compliance/sit-get-started-exact-data-match-create-rule-package?view=o365-worldwide#create-a-rule-package-manually for instructions to configure an Exact Data Match sensitive information type using XML and PowerShell. 

Reviewer: ```suggestion
- If the EDM SIT detects content when secondary evidence is not required but fails to detect content when secondary evidence requirements are included, or detects fewer matches than expected, make sure values for secondary evidence you are trying to detect are composed of a single "token" (a word or string that does not contain spaces in it). For example, first or last names can be single or multi-token strings since each can be composed of multiple words. When matching additional evidence EDM will, by default, match individual tokens in the text, so multi-token strings will not be detected. To address this, you can define a sensitive information type that finds matches to such multi-token strings (for example, using a regular expression such as \b[A-Z][a-z]{1,25}([ -][A-Z][a-z]{1,25}){0,4}\b which will match any sequence of one to five consecutive words that start with an uppercase character), and use this sensitive information type as the "Classification" element for the additional evidence conditions in your EDM sensitive info type XML using the same syntax used for the primary element. See [Create a rule package manually](/microsoft-365/compliance/sit-get-started-exact-data-match-create-rule-package?view=o365-worldwide#create-a-rule-package-manually) for instructions to configure an exact data match sensitive information type using XML and PowerShell. 
```

Author: Hi. I see one more typo in my original that wasn't fixed, but not sure how to fix it without breaking this commit. It is the word "one0". Can @JohanFreelancer9 fix it or let me know how I fix it myself without breaking this commit?

Reviewer: Thank you @E-Saggese, suggested change updated.
